docs/operations/api-reference.md

```diff
@@ -96,11 +96,11 @@ Returns the percentage of segments actually loaded in the cluster versus segment
 
  * `/druid/coordinator/v1/loadstatus?simple`
 
-Returns the number of segments left to load until segments that should be loaded in the cluster are available for queries. This does not include replication.
+Returns the number of segments left to load until segments that should be loaded in the cluster are available for queries. This does not include replica of segments.
```

Comment:
"This does not include replica of segments." -> "This does not include segment replication counts."

Reply:
Done